docs/intro/tutorial01.txt

```diff
@@ -430,7 +430,7 @@ look like this:
         'django.contrib.messages',
         'django.contrib.staticfiles',
         'polls',
```

Comment:
No, nothing wrong with them. I was just surprised to see this after reading the django-developers thread, since half of the impetus for this change is to prevent the missing trailing commas problem with tuples. Just a minor observation. Fine either way.